.circleci/config.yml

```diff
@@ -370,6 +372,114 @@ jobs:
     environment:
       TOXENV: py37-integration-ethtester
       ETHEREUM_TESTER_CHAIN_BACKEND: eth_tester.backends.PyEVMBackend
+  #
+  # Python 3.8
+  #
+  py38-core:
+    <<: *common
+    docker:
+      - image: circleci/python:3.8
+    environment:
+      TOXENV: py38-core
+
+  py38-ens:
+    <<: *common
+    docker:
+      - image: circleci/python:3.8
+    environment:
+      TOXENV: py38-ens
+
+  py38-ethpm:
+    <<: *ethpm_steps
+    docker:
+      - image: circleci/python:3.8
+    environment:
+      TOXENV: py38-ethpm
+      WEB3_INFURA_PROJECT_ID: ae13436d29b34850b640af9c80f80871
+      WEB3_INFURA_API_SECRET: 067ddad6021048f0840a38a4e52eb568
+
+  py38-integration-goethereum-ipc-1.7.2:
+    <<: *geth_steps
+    docker:
+      - image: circleci/python:3.8
+    environment:
+      TOXENV: py38-integration-goethereum-ipc
+      GETH_VERSION: v1.7.2
+
+  py38-integration-goethereum-http-1.7.2:
+    <<: *geth_steps
+    docker:
+      - image: circleci/python:3.8
+    environment:
+      TOXENV: py38-integration-goethereum-http
+      GETH_VERSION: v1.7.2
+
+  py38-integration-goethereum-ws-1.7.2:
+    <<: *geth_steps
+    docker:
+      - image: circleci/python:3.8
+    environment:
+      TOXENV: py38-integration-goethereum-ws
+      GETH_VERSION: v1.7.2
+
+  py38-integration-goethereum-ipc-1.8.22:
+    <<: *geth_steps
+    docker:
+      - image: circleci/python:3.8
+    environment:
+      TOXENV: py38-integration-goethereum-ipc
+      GETH_VERSION: v1.8.22
+
+  py38-integration-goethereum-http-1.8.22:
+    <<: *geth_steps
+    docker:
+      - image: circleci/python:3.8
+    environment:
+      TOXENV: py38-integration-goethereum-http
+      GETH_VERSION: v1.8.22
+
+  py38-integration-goethereum-ws-1.8.22:
+    <<: *geth_steps
+    docker:
+      - image: circleci/python:3.8
+    environment:
+      TOXENV: py38-integration-goethereum-ws
+      GETH_VERSION: v1.8.22
+
+  py38-integration-parity-ipc:
+    <<: *parity_steps
+    docker:
+      - image: circleci/python:3.8
```

Comment:
I want to point out that it seems like stretch images are no longer needed. Back when [this commit](https://github.com/ethereum/web3.py/commit/8327480b0f1408c59050e243d8b664b065d4c2c4) was added parity would fail to build on other distros.